src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -608,6 +629,14 @@ int UProbeManager::DeployOpenSSLUProbes(const absl::flat_hash_set<md::UPID>& pid
       continue;
     }
 
+    PX_ASSIGN_OR(const auto exe_path, proc_parser_->GetExePath(pid.pid()), continue);
+
+    if (std::find(uprobe_opt_out_.begin(), uprobe_opt_out_.end(), exe_path.filename()) !=
```

Comment:
.contains() if you are able to switch to `absl::flat_hash_map`

Reply:
See https://github.com/pixie-io/pixie/pull/1971/commits/1207906d583090c861d06971e20a6da9c38d69fd